scripts/configure-webrtc.sh

```diff
@@ -8,4 +8,10 @@ export PATH=$DEPOT_TOOLS:$PATH
 
 cd ${SOURCE_DIR}
 
+if [ "$TARGET_ARCH" == "arm" ]; then
+    python build/linux/sysroot_scripts/install-sysroot.py --arch=arm
+else
+    python build/linux/sysroot_scripts/install-sysroot.py --arch=amd64
```

Comment:
Looks like this was not called before. Is it actually necessary for the non-ARM case?

Reply:
Actually, it looks like a similar line is in `scripts/download-webrtc.sh`. Should both of these move there?

Reply:
> Looks like this was not called before. Is it actually necessary for the non-ARM case?
It was called from download-webrtc.sh, but we should call this from configure-webrtc.sh.
If you run "TARGET_ARCH=arm npm i" second time, process will not call download-webrtc.sh because it is already downloaded.

Reply:
> Actually, it looks like a similar line is in `scripts/download-webrtc.sh`. Should both of these move there?

It is moved to here.